src/wallet/rpcwallet.cpp

```diff
@@ -2202,6 +2202,8 @@ UniValue reservebalance(const UniValue& params, bool fHelp)
 // presstab HyperStake
 UniValue setstakesplitthreshold(const UniValue& params, bool fHelp)
 {
+    std::string strComment="";
```

Comment:
nit: move this initialization further down so as to be just above where it would actually be used. No sense initializing it if the input command isn't valid, is only asking for the help message, or if the threshold is invalid.

Reply:
done